src/profile/components/deleteProfile/DeleteProfile.tsx

```diff
@@ -4,34 +4,50 @@ import { loader } from 'graphql.macro';
 import { useTranslation } from 'react-i18next';
 import * as Sentry from '@sentry/browser';
 import { Checkbox } from 'hds-react';
+import { useHistory } from 'react-router';
+import { useMatomo } from '@datapunt/matomo-tracker-react';
 
 import ConfirmationModal from '../modals/confirmationModal/ConfirmationModal';
 import ExpandingPanel from '../../../common/expandingPanel/ExpandingPanel';
 import Button from '../../../common/button/Button';
-import { ServiceConnectionsQuery } from '../../../graphql/generatedTypes';
+import { ServiceConnectionsQuery as ServiceConnectionsRoot } from '../../../graphql/generatedTypes';
 import useToast from '../../../toast/useToast';
 import styles from './deleteProfile.module.css';
+import useDeleteProfile from '../../../gdprApi/useDeleteProfile';
+import checkBerthError from '../../helpers/checkBerthError';
+import BerthErrorModal from '../modals/berthError/BerthErrorModal';
```

Comment:
I wonder what all these \*berth\* things are... But I see things were just moved around, so no need to do anything about that in this PR. But perhaps need to write it down somewhere that there could be something that should be checked if it's still correct (as there are and will be other services than just berth).

Reply:
Good point!